src/client/mod.rs

```diff
@@ -390,7 +389,7 @@ impl KafkaClient {
     pub fn new(hosts: Vec<String>) -> KafkaClient {
         KafkaClient {
             config: ClientConfig {
-                client_id: CLIENTID.to_owned(),
+                client_id: String::new(),
```

Comment:
What would you think about using `"kafka-rust"` as a default, rather than no client ID?